### DISCUSSION TITLE
Do i need polyfill for :has selector ?

### DISCUSSION THREAD

**Discussion by issam-seghir** [csstools/postcss-plugins#1142] [Q&A]
Do i need polyfill for `:has` selector ?

**Comment by romainmenke** [comment]
Hi @issam-seghir,

Can you provide a bit more info?

In general you require a polyfill only when :
- you actually use the feature
- your project breaks in a bad way in browser versions that do not support the feature
- your target audience uses such older browser versions

None of those things can be known by anyone other than yourself :)

**Reply by romainmenke** [reply]
Also, it is helpful if you phrase your question in a way that helps us give you an answer :)

For your specific question we can only answer with :
- `yes`
- `no`
- `we don't understand what you are asking`

None of those replies are useful to you ;)

---------

Take the time to fully describe :
- what you are doing
- what your expected outcome is
- what you get instead
- why the actual outcome is an issue

**Comment by issam-seghir** [comment]
> Hi @issam-seghir,
> 
> Can you provide a bit more info?
> 
> In general you require a polyfill only when :
> 
> * you actually use the feature
> * your project breaks in a bad way in browser versions that do not support the feature
> * your target audience uses such older browser versions
> 
> None of those things can be known by anyone other than yourself :)

I definitely use it and I target modern browsers. However, it seems that it is not compatible with Firefox at the moment. so i think its required for me thanks
